app.js

```diff
@@ -1,8 +1,119 @@
-// import functions and grab DOM elements
+import { renderPoll } from './render.js';
+
+const form = document.getElementById('form');
+const optionButtonOne = document.querySelector('#option-one-vote');
+const optionButtonTwo = document.querySelector('#option-two-vote');
+const optionMinusButtonOne = document.querySelector('#option-one-minus');
+const optionMinusButtonTwo = document.querySelector('#option-two-minus');
+const previousPollsEl = document.querySelector('.past-poll-results');
+const endPollButton = document.getElementById('poll-end');
+const pollQuest = document.getElementById('question-result');
+const optOne = document.getElementById('option-one-results');
+const optTwo = document.getElementById('option-two-results');
+const optOneVotes = document.getElementById('option-one-vote-results');
+const optTwoVotes = document.getElementById('option-two-vote-results');
+const startPollButton = document.getElementById('start-poll');
+
+//state 
+
+let oneVotes = 0;
+let twoVotes = 0;
+let question = '';
+let optionOne = '';
+let optionTwo = '';
+
+const pastArray = [];
+
+
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    
+    const formData = new FormData(form);
+    
+    question = formData.get('poll-question');
+    optionOne = formData.get('option-one');
+    optionTwo = formData.get('option-two');
+    
+    displayCurrentPoll();
+});
+
+optionButtonOne.addEventListener('click', () => {
+    oneVotes++;
+
+    optOneVotes.textContent = oneVotes;
```

Comment:
Remember, for each add and subtract vote button event listener we have that `displayCurrentPoll()` function that does the DOM updating for us, no need to go through each and reset the textContent for each display element when we can just call our function.